attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>nit 1: Equation in 2 would be easier to read if formatted in a single line.</p><p>nit 2: you mention St-1 in the equation but doesn't explicit say this is the RNN state.</p><p>nit 3: mention that this attention schema is called "dot product attention"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>